packages/storefront/src/components/PropsTable.vue

```diff
@@ -0,0 +1,155 @@
+      return unorderedPropsMeta
+        ? Object.keys(unorderedPropsMeta)
+            .sort()
+            .reduce((result, key) => ({ ...result, [key]: unorderedPropsMeta[key] }), {})
+        : {};
+    }
+
+    public get hasBreakpointCustomizableProp(): boolean {
+      return this.propsMeta
+        ? Object.values(this.propsMeta).some((propMeta) => propMeta.isBreakpointCustomizable)
+        : false;
+    }
+
+    public formatDescription(meta: PropMeta): string {
+      return meta.description
+        ? meta.description
+            .replace(/@(deprecated)/, '<strong class="deprecated">$1</strong>') // deprecated annotation
+            .replace(/`(.+?)`/g, (_, g1) => `<code>${formatHtml(g1)}</code>`) // prop references in backticks
+        : '';
+    }
+
+    public formatType(meta: PropMeta): string {
+      return Array.isArray(meta.allowedValues) || meta.isBreakpointCustomizable
+        ? [
+            ...(meta.type.includes('|')
+              ? [] // inline union type is taken care of via allowedValues
+              : [
+                  meta.type !== 'string' && meta.type !== 'number' && meta.type !== 'boolean'
+                    ? `type ${meta.type} =` // literal types, etc.
+                    : meta.type, // simple type
+                ]),
+            ...(Array.isArray(meta.allowedValues)
+              ? meta.allowedValues.map(
+                  (val) =>
+                    wrapInCodeCode(
+                      val === 'string'
+                        ? val
+                        : val === null
+                          ? 'undefined'
+                          : typeof val === 'number'
+                            ? `${val}`
+                            : `'${val}'`
+                    ) + (meta.deprecatedValues?.includes(val) ? '<span title="deprecated"> 🚫</span>' : '')
+                )
+              : []),
+            ...(meta.isBreakpointCustomizable ? [`BreakpointCustomizable<${meta.type}>`] : []),
+          ]
+            // allowedValues are already wrapped with code tag because of trailing deprecated icon, but others are not
+            .map((item) => (item.match(/<code>.+?<\/code>/) ? item : wrapInCodeCode(item)))
+            .join('<br>\n')
+        : meta.isAria && typeof meta.allowedValues === 'object'
+          ? // aria props
+            wrapInCodeCode(
+              `type ${meta.type} = {
+${Object.entries(meta.allowedValues)
+  // possible values are output as string, even though actual types are more precise
+  .map(([key, val]) => `&nbsp;&nbsp;'${key}'?: ${val};`)
+  .join('\n')}
+}`
+            )
+          : wrapInCodeCode(meta.type); // all other cases
+    }
+  }
+</script>
+
+<style scoped lang="scss">
+  @use '@porsche-design-system/components-js/styles' as *;
+  @import '../styles/shared.styles';
+
+  :deep(.deprecated) {
+    color: #d5001c;
+    text-transform: uppercase;
+  }
+
+  table,
+  :deep(table) {
```

Comment:
why do we need `:deep` here?

Reply:
To apply styling on nested `code` tags that are not rendered via template, but instead `v-html`.
```
@mixin tableStyles {
  margin-top: $pds-spacing-fluid-medium;
  border-collapse: collapse;

  code ~ code::before { // <---
    content: '| ';
  }
  ...
```